homeassistant/components/iron_os/entity.py

```diff
@@ -0,0 +1,45 @@
+"""Base entity for Pinecil integration."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+from homeassistant.helpers.device_registry import CONNECTION_BLUETOOTH, DeviceInfo
+from homeassistant.helpers.entity import EntityDescription
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import MANUFACTURER, MODEL
+from .coordinator import PinecilCoordinator
+
+
+class PinecilBaseEntity(CoordinatorEntity[PinecilCoordinator]):
+    """Base Pinecil entity."""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        coordinator: PinecilCoordinator,
+        entity_description: EntityDescription,
+        context: Any = None,
```

Comment:
why the context?

Reply:
Leftover from settings entities, will remove it for now and re-implement it again later in another PR when I need it for the settings entities.